src/python/pants/backend/adhoc/code_quality_tool.py

```diff
@@ -178,12 +181,17 @@ async def find_code_quality_tool(request: CodeQualityToolAddressString) -> CodeQ
         AddressInput,
         AddressInput.parse(request.address, description_of_origin="code quality tool target"),
     )
-
     addresses = Addresses((tool_address,))
     addresses.expect_single()
 
-    tool_targets = await Get(Targets, Addresses, addresses)
-    target = tool_targets[0]
+    wrapped_target = await Get(
+        WrappedTarget,
+        WrappedTargetRequest(
+            address=tool_address,
+            description_of_origin=f"the configured `target` for Code quality tool {request.description_of_origin}",
+        ),
+    )
+    target = wrapped_target.target
```

Comment:
The changes in this file are only to give a better error message when a user mistypes the address of a code quality tool target.